pkg/controller/legacy/cache.go

```diff
@@ -962,6 +975,9 @@ func (c *k8scache) SwapChangedObjects() *convtypes.ChangedObjects {
 	for _, ep := range ch.EndpointsNew {
 		addChanges(convtypes.ResourceEndpoints, eventUpdate, ep.Namespace, ep.Name)
 	}
+	for _, eps := range ch.EndpointSlicesUpd {
+		addChanges(convtypes.ResourceEndpoints, eventUpdate, eps.Namespace, eps.Annotations["kubernetes.io/service-name"])
+	}
```

Comment:
This should be `eps.Labels`, sorry about my previous comment when I suggested annotation instead.